Kudu.Core/Deployment/Generator/ExternalCommandFactory.cs

```diff
@@ -87,7 +87,8 @@ public Executable BuildCommandExecutable(string commandPath, string workingDirec
                 Path.GetDirectoryName(PathUtility.ResolveMSBuildPath()),
                 Path.GetDirectoryName(PathUtility.ResolveGitPath()),
                 Path.GetDirectoryName(PathUtility.ResolveVsTestPath()),
-                Path.GetDirectoryName(PathUtility.ResolveSQLCmdPath())
+                Path.GetDirectoryName(PathUtility.ResolveSQLCmdPath()),
+                Path.GetDirectoryName(PathUtility.ResolveFSharpCPath())
```

Comment:
I also added support for F# scripts `*.fsx`  as webjobs, which I was using for testing, and that requires the path to `fsi.exe` on the path. I can remove it, but I figured it why not have it.